src/chkstat.cpp

```diff
@@ -1058,7 +1059,12 @@ bool Chkstat::safeOpen(EntryContext &ctx)
         if (is_parent_element)
         {
             // either owned by root or by our effective UID
```

Comment:
The comment could describe the following line more clearly:
"neither owned by root nor by our effective UID"